Library/Source/RuntimeAndroid.cpp

```diff
@@ -16,6 +17,18 @@ namespace babylon
         // android stub
     }
 
+    void RuntimeAndroid::SetWindow(ANativeWindow* nativeWindowPtr)
```

Comment:
What is the reason for a separate function here? Can it not go in the constructor?

Reply:
No, when there is a screen rotation/app switching, the window is destroyed and a new one is created.

Reply:
Interesting. We should talk about the contract for this. Right now, it's not clear (at least to me) what you are supposed to do with this function.